src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/UpdateGithubIssueTest.cs

```diff
@@ -0,0 +1,131 @@
+            secondError.Object.Success = false;
+            secondError.Object.ErrorMessage = "build: 14	Unexpected error processing action: Validation Failed";
+            secondError.Object.Action = "Updating existing issue";
+
+            Context.RepoBranchUpdateInMemory = new List<RepositoryBranchUpdateHistoryEntry>
+                {firstError.Object , secondError.Object};
+            Mock<Maestro.Data.Models.Subscription> subscription = new Mock<Maestro.Data.Models.Subscription>();
+            subscription.Object.Id = Guid.Parse("ee8cdcfb-ee51-4bf3-55d3-08d79538f94d");
+            subscription.Object.SourceRepository = "Source Repo";
+            subscription.Object.TargetRepository = "Target Repo";
+            Context.Subscriptions.Add(subscription.Object);
+            Context.SaveChanges();
+
+            Mock<Octokit.Repository> repository = new Mock<Repository>();
+            GithubClient.Setup(x => x.Repository.Get(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(repository.Object);
+            Issue issueCreated = GetIssue();
+            List<NewIssue> newIssues = new List<NewIssue>();
+            GithubClient.Setup(x => x.Issue.Create(It.IsAny<long>(), Capture.In(newIssues))).ReturnsAsync(issueCreated);
+            GithubClient.Setup(x => x.Issue.Get(0, 1)).ReturnsAsync(issueCreated);
+            List<IssueUpdate> issueUpdate = new List<IssueUpdate>();
+            GithubClient.Setup(x => x.Issue.Update(0, 1, Capture.In(issueUpdate))).ReturnsAsync(issueCreated);
+            DependencyUpdateErrorProcessor errorProcessor =
+                ActivatorUtilities.CreateInstance<DependencyUpdateErrorProcessor>(Scope.ServiceProvider,
+                    Context);
+            await errorProcessor.ProcessDependencyUpdateErrorsAsync();
+            string issueBody = $@"The following errors have been detected when attempting to update dependencies in 
+'https://github.com/maestro-auth-test/maestro-test2'
+
+ [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**SubscriptionId:** 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+**Source Repository :**  'Source Repo'
+**Target Repository :**  'Target Repo'
+**Branch Name :**  '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :** 'UpdateAssetsAsync'
+**Action :** 'Creating new issue'
+**Last seen :** '1/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            string updatedIssueBody =
+                $@"The following errors have been detected when attempting to update dependencies in
+'https://github.com/maestro-auth-test/maestro-test2'
+
+  [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
```

Comment:
This "ee8" number is all over the place in this file.   Make a private constant in here and use it.  that's actually probably true of most of the strings that are set and then asserted... easier to tell the set and the assert are meant to be the same thing if they are just referencing some constant. Then instead of trying to figure out what "ee8cd...." is referring to, these tests will use some variable like "MockSubscriptionId".

Reply:
That's probably true of most of the strings and numbers.  If they are used a lot, make them consts in the class, if they are only used in a single test, local consts in the method.